### ISSUE TITLE
[Bug] Can't pip install from github

### ISSUE THREAD

**Issue by austinmw** [open-mmlab/mmocr#1783]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmocr/issues) and [Discussions](https://github.com/open-mmlab/mmocr/discussions) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (0.x)](https://github.com/open-mmlab/mmocr) or [latest version (1.x)](https://github.com/open-mmlab/mmocr/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

1.x branch https://github.com/open-mmlab/mmocr/tree/dev-1.x

### Environment

I'm attempting to create the following conda environment via `mamba env create -f environment.yml`:

`environment.yml`:
```yaml
name: openmmlab
channels:
  - conda-forge
  - defaults
  - pytorch
  - nvidia
dependencies:
  - python=3.9.16
  - ipython=8.4.0
  - libjpeg-turbo=2.1.*
  - ipykernel=6.19.*
  - nb_conda_kernels=2.3.*
  - pytorch=1.13.1
  - pytorch-cuda=11.7
  - torchvision=0.14.1
  - tree
  - pip
  - pip:
    - openmim==0.3.6
    - tqdm==4.65.0
    - pandas==1.5.3
    - ipywidgets==7.7.2
    - sagemaker==2.138.0
    - gitpython==3.1.31
    - git+https://github.com/open-mmlab/mmengine.git@v0.6.0#egg=mmengine
    - git+https://github.com/open-mmlab/mmcv.git@v2.0.0rc4#egg=mmcv
    - git+https://github.com/open-mmlab/mmclassification.git@v1.0.0rc5#egg=mmcls
    - git+https://github.com/open-mmlab/mmdetection.git@v3.0.0rc6#egg=mmdet
    - git+https://github.com/open-mmlab/mmsegmentation.git@v1.0.0rc6#egg=mmsegmentation
    - git+https://github.com/open-mmlab/mmocr.git@v1.0.0rc6#egg=mmocr
```

### Reproduces the problem - code sample

See below

### Reproduces the problem - command or script

I run `mamba env create -f environment.yml` on the above conda environment file

### Reproduces the problem - error message

```
Looking in indexes: https://pypi.org/simple, https://pip.repos.neuron.amazonaws.com
Collecting git+https://github.com/open-mmlab/mmocr.git@v1.0.0rc6
  Cloning https://github.com/open-mmlab/mmocr.git (to revision v1.0.0rc6) to /tmp/pip-req-build-f78_6d5e
  Running command git clone --filter=blob:none --quiet https://github.com/open-mmlab/mmocr.git /tmp/pip-req-build-f78_6d5e
  Resolved https://github.com/open-mmlab/mmocr.git to commit bfb36d81b3c87a9e4bb04832d8e9d813e55e07a3
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [1 lines of output]
      error in mmocr setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

### Additional information

The other OpenMMLab installs listed in the environment file complete successfully. I haven't been able to determine what is different about MMOCR.

**Comment by gaotongxiao** [comment]
Hi, I just re-installed MMOCR from github but didn't run into any errors:

`pip install git+https://github.com/open-mmlab/mmocr.git@v1.0.0rc6#egg=mmocr`

I'm not quite familiar with `mamba`. Was it doing something different for installation?

**Comment by VanderBieu** [comment]
I have encountered the same issue with the following command
`pip install -v -e .`

**Comment by gaotongxiao** [comment]
@VanderBieu Could you open up a new issue and elaborate the context?

**Comment by VanderBieu** [comment]
> @VanderBieu Could you open up a new issue and elaborate the context?

Sure.

**Comment by gaotongxiao** [comment]
@ssnseawolf I was initially puzzled by the "missing dependency" issue, as MMOCR has never required CLIP as a dependency.

Fortunately, I discovered that the problem stemmed from a syntax error in requirements/mminstall.txt. This issue has been resolved in PR #1781, and the latest dev-1.x branch should now function seamlessly.

We're planning to release version 1.0.0 soon, which will also include this fix.
